javascript/GridField.js

```diff
@@ -146,6 +152,37 @@
 			}
 		});
 
+		/**
+		 * Don't allow users to submit empty values in grid field auto complete inputs.
+		 */
+		$('.ss-gridfield .add-existing-autocompleter').entwine({
+			onbuttoncreate: function () {
+				var self = this;
+
+				this.toggleDisabled();
+
+				this.find('input[type="text"]').bind('keyup', function () {
```

Comment:
I think we should use 'on' instead of 'bind', as it is more ajax friendly.

https://api.jquery.com/on/